test/integration/ClientQueryTest.java

```diff
@@ -53,6 +56,7 @@ public static void setUpClass() throws InterruptedException, IOException, Timeou
         grakn = new GraknCoreRunner();
         grakn.start();
         graknClient = GraknClient.core(grakn.address());
+        if (graknClient.databases().contains("grakn")) graknClient.databases().delete("grakn");
```

Comment:
This change allows this test to be run and succeed against a local instance of Grakn.

Reply:
nice!